### PR TITLE
cckit lab

### PR DESCRIPTION
**Pull request by vitiko**
Signed-off-by: vitiko <vitiko@mail.ru>

## Checklist for Hyperledger Lab Proposal PR

- [ ] Read [README.md](https://github.com/hyperledger-labs/hyperledger-labs.github.io/blob/master/README.md) carefully
- [ ] Make sure your changes are committed with the proper sign-off
- [ ] Use the labs name as the PR title
- [ ] Copy the short description of the proposal as the description of the PR
- [ ] Remove this checklist section
- [ ] Submit your PR and the labs stewards will review your proposal
- [ ] Ask your sponsor to confirm sponsorship by stating so in a comment to the PR

### CODE REVIEW

File: labs/cckit.md [hunk 1]
```diff
@@ -0,0 +1,49 @@
...
+parent: Labs
+---
+
+# Lab Name
+
+[CCKit](https://github.com/hyperledger-labs/cckit)
+
+# Short Description
+
+Programming framework for building Hyperledger Fabric Golang on-chain (chaincode) and off-chain applications&
```
Comment (tkuhrt) [inline labs/cckit.md:13]:
```suggestion
Programming framework for building Hyperledger Fabric Golang on-chain (chaincode) and off-chain applications
```

File: labs/cckit.md [hunk 2]
```diff
@@ -0,0 +1,49 @@
...
+- https://github.com/kolayuk
+- https://github.com/stepanshurkov
+
+# Sponsor
+
+- https://github.com/C0rWin - HLF maintainer
+
+# Pre-existing repository
+
+- https://github.com/s7techlab/cckit
```
Comment (tkuhrt) [inline labs/cckit.md:48]: It looks like some of the commits in this repository do not contain DCO signoff. Hyperledger requires that all contributed code has a DCO sign off. Looking at these repositories, it appears that this is not the case. There are two options that we normally consider for fixing this:

1. Amend existing commits in the repository to include DCO sign off
1. Manually transfer the code from the existing repo to the newly created repo within the hyperledger-labs

Secondly, it appears that this code is currently MIT licensed. According to [Hyperledger's charter](https://hyperledger.org/about/charter):
> Members agree that all new inbound code contributions to HLP shall be made under the Apache License, Version 2.0 (available at http://www.apache.org/licenses/LICENSE-2.0). All contributions shall be accompanied by a Developer Certificate of Origin sign-off (http://developercertificate.org/) that is submitted through a Governing Board and LF-approved contribution process. Such contribution process will include steps to also bind non-Member Contributors and, if not self-employed, their employer, to the licenses expressly granted in the Apache License, Version 2.0 with respect to such contribution.

**Review by tkuhrt — CHANGES_REQUESTED** [review]
This is a good lab; however, we will need to make sure that the IP considerations from the charter are resolved.

File: labs/cckit.md (hunk 2 above)
Comment (vitiko) [reply labs/cckit.md:48]: Hi, @tkuhrt ! Thank you for comments.  
The license issue is [fixed](https://github.com/s7techlab/cckit/blob/master/LICENSE).  
Transferring the code - I prefer the second option (manually transfer the code from the existing repo to the newly created repo within the hyperledger-labs)

**Review by tkuhrt — APPROVED** [review]

**Comment by C0rWin** [comment]
Not sure it requires, given @tkuhrt already approved. I also believe this is an excellent lab proposal; therefore, I support it and will be glad to see it among other lab projects.

**Review by C0rWin — APPROVED** [review]

**Review by lehors — APPROVED** [review]
LGTM!

**Pull request merged**